### ISSUE TITLE
System - Current user locks are incompatible with task locks

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#269]
User locks are simply flags set on data records. These flags are not honored by background tasks. User locks, if still needed, should be changed to the same type of lock used by tasks.

**Comment by JoshuaSBrown** [comment]
So this is a dead feature that needs to be removed from the UI?

**Comment by dvstans** [comment]
No, not dead! Locks have important use cases. They need to be fixed.
